accounting/account/serializers.py

```diff
@@ -44,7 +44,7 @@ class AccountSerializer(UuidModelSerializer):
     class Meta(UuidModelSerializer.Meta):
         model = models.Account
         fields = (UuidModelSerializer.Meta.fields +
-                  ('user', 'address', 'business_name', 'occupation', 'bank_accounts', 'vat',))
+                  ('user', 'address', 'business_name', 'occupation', 'bank_account', 'vat',))
```

Comment:
You have to update the `bank_accounts` above to be `bank_account` too.

Reply:
@clemente Shoot, one more change I had to do locally: the serializer cannot use `many=True` since it's just 1-to-1 now.

Reply:
Both fixed, thanks.